### PR TITLE
lookup, v1alpha1, grpc types

### PR DESCRIPTION
**Pull request by jzelinskie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,98 +1,61 @@
...
 
-class Document(ArrakisNamespace):
-    """ A model to represent documents in your business domain (if you have such a thing). """
+client = Client("grpc.authzed.com:443", bearer_token_credentials("mytoken"))
+resp = client.ReadSchema(ReadSchemaRequest(object_definitions_names=["example/user"]))
+print(resp.object_definitions)
+```
 
-    __namespace__ = "yourtenant/document"
+If an event loop is running when the client is initialized, RPCs are async:
```
Comment (josephschorr) [inline README.md:38]: `RPCs` -> all API function calls

Comment (josephschorr) [reply README.md:38]: Add a comment on how to make this explicit

File: authzed/api/v0/__init__.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    "ExpandResponse",
+    "LookupRequest",
+    "LookupResponse",
+    "ReadRequest",
+    "ReadResponse",
+    "RelationTupleFilter",
+    "WriteRequest",
+    "WriteResponse",
+    # Developer Service
+    "EditCheckRequest",
```
Comment (josephschorr) [inline authzed/api/v0/__init__.py:118]: Not sure we need to expose the developer service

File: examples/arrakisclient/README.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+A complete version of this example can be found in this repository in [`basic.py`](basic.py)
+
+In Python, we have support for declaring ORM-like models which allow you to easily interact with the data in your tenant.
+Here we define some models for `User` and `Document` which match the tenant namespace configuration.
+
+```py
+from arrakisclient.types.namespace import ArrakisNamespace, Relation
+
+
+# Define some ORM-like models, must match the namespaces configured in the tenant
```
Comment (josephschorr) [inline examples/arrakisclient/README.md:10]: Use the new terminology here and add a note of why the classes are named differently

Comment (jzelinskie) [reply examples/arrakisclient/README.md:10]: This just got moved and I'm not sure it's a totally valid example. Maybe we should just flag everything as arrakisclient as legacy and move on?

File: authzed/api/v0/__init__.py (hunk 2 above)
Comment (jzelinskie) [reply authzed/api/v0/__init__.py:118]: I think it's fine to expose these in case we need it for ourselves.

**Review by josephschorr — APPROVED** [review]
LGTM

**Pull request merged**